Latest meta-mono breaks building msbuild-native

At the beginning of `do_compile` this is printed:
```
DEBUG: Executing shell function do_compile
** Downloading MSBUILD from https://github.com/mono/msbuild/releases/download/0.08/mono_msbuild_6.4.0.208.zip
cp: cannot stat ‘/data/OE/sicom-manifest-3.2/tmp-sicom-glibc/work/x86_64-linux/msbuild-native/16.6-r0/recipe-sysroot-native/usr/lib/libhostfxr.so’: No such file or directory
```
and the log ends with:
```
Build FAILED.

/data/OE/sicom-manifest-3.2/tmp-sicom-glibc/work/x86_64-linux/msbuild-native/16.6-r0/mono-nuget-packages/microsoft.dotnet.arcade.sdk/1.0.0-beta.20202.9/tools/Tools.proj : error MSB4242: The SDK resolver "Microsoft.DotNet.MSBuildSdkResolver" failed to run. hostfxr assembly:<unknown assembly> type:<unknown type> member:(null)
/data/OE/sicom-manifest-3.2/tmp-sicom-glibc/work/x86_64-linux/msbuild-native/16.6-r0/git/artifacts/mono-msbuild/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.ILLink.targets(14,3): error MSB4242: The SDK resolver "Microsoft.DotNet.MSBuildSdkResolver" failed to run. hostfxr assembly:<unknown assembly> type:<unknown type> member:(null)
```
The new `msbuild-libhostfxr` recipe doesn't install `libhostfxr.so` to `${libdir}`.

--- a/recipes-mono/msbuild/msbuild-libhostfxr_3.1.11.bb
+++ b/recipes-mono/msbuild/msbuild-libhostfxr_3.1.11.bb
@@ -53,6 +53,10 @@ do_install() {
 	install -m755 ${B}/cli/fxr/libhostfxr.so ${D}${libdir}/mono/msbuild/Current/bin/SdkResolvers/Microsoft.DotNet.MSBuildSdkResolver/libhostfxr.so
 }
 
+do_install_append_class-native () {
+	install -m755 ${B}/cli/fxr/libhostfxr.so ${D}${libdir}
+}
+
 FILES_${PN} = " \
     ${libdir}/mono/msbuild/Current/bin/SdkResolvers/Microsoft.DotNet.MSBuildSdkResolver/libhostfxr.so \
 "

